_posts/2022-04-11-1.15-announcement.md

```diff
@@ -0,0 +1,375 @@
+like to go into two in more detail.
+
+
+### CAST/Type system enhancements
+
+Data appears in all sorts and shapes but is often not in the type that you need 
+it to be. That’s why 
+[casting](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/dev/table/types/#casting) 
+is one of the most common operations in SQL. In Flink 
+1.15, the default behavior of a failing CAST has changed from returning a null to 
+returning an error. The new behavior is more compliant with the SQL standard. The old 
+casting behavior can still be used by calling the newly introduced TRY_CAST function. 
+Next to this change, many bugs have been fixed and improvements made to the casting 
+functionality, to make sure that you’re getting correct results. The old CAST behavior 
+can be restored via a configuration flag. The effort is very well illustrated in this 
+JIRA issue. See the Flink documentation for details. 
+
+
+### JSON functions
+
+JSON is one of the most popular data formats and SQL users increasingly need to build 
+and read these data structures.  Multiple 
+[JSON](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/dev/table/functions/systemfunctions/#json-functions)
+functions have been added to Flink SQL 
+according to the SQL 2016 standard. It allows users to inspect, create and modify JSON 
+strings using the Flink SQL dialect.
+
+
+## Community enablement
+
+Enabling people to build streaming data pipelines to solve their use cases: That’s 
+what the Apache Flink community wants to do. The community is well aware that a 
+technology like Apache Flink is never used on its own and will always be part of a 
+bigger architecture. It is important to operate well on clouds, it is important to 
+connect to other systems, it is important to make the programming languages Apache 
+Flink supports work. Here’s what has been done in that regard.
+
+
+### Cloud Interoperability
+
+There are users operating Flink deployments in a cloud infrastructure of various 
+cloud providers, there are also services that offer to manage Flink deployments for 
+users on them. With Flink 1.15 an recoverable writer for Google Cloud Storage has 
+been added. Tidying up the connectors was also in scope and there has been work 
+especially on connectors for the AWS ecosystem. (e.g. 
+[KDS](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/kinesis/), 
+[Firehose](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/firehose/))
+
+
+### ElasticSearch sink
+
+As said, there was significant work on Flink’s connector ecosystem; still we want 
+to mention the 
+[ElasticSearch sink](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/elasticsearch/)
+at this point, because it was implemented following 
+very closely the new connector interfaces offering asynchronous functionality as well 
+as end-to-end semantics. This sink will act as a role model going forward.
```

Comment:
```suggestion
### The Elasticsearch sink

There was significant work on Flink’s overall connector ecosystem, but we want to highlight the 
[Elasticsearch sink](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/connectors/datastream/elasticsearch/) because it was implemented with 
the new connector interfaces, which offers asynchronous functionality coupled with end-to-end semantics. 
This sink will act as a template to follow in the future.
```